### PR TITLE
Cluster Lifecycle Controller integration

### PR DESCRIPTION
**Pull request by aermakov-zalando**
* Create IAM role for the Cluster Lifecycle Controller
 * Configure the replacement strategy by setting the labels
 * If `experimental_cluster_lifecycle_controller` is set to `true`, run the controller

### CODE REVIEW

File: cluster/manifests/cluster-lifecycle-controller/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-lifecycle-controller
+  namespace: kube-system
+  labels:
+    application: cluster-lifecycle-controller
+    version: pr-1-39
```
Comment (aermakov-zalando) [inline cluster/manifests/cluster-lifecycle-controller/deployment.yaml]: **TBD**: change this.

**Comment by aermakov-zalando** [comment]
👍

**Comment by aermakov-zalando** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**